src/main/java/solvd/laba/ermakovich/hu/mongo/DoctorRepository.java

```diff
@@ -1,18 +1,18 @@
-package solvd.laba.ermakovich.hu.repository;
+package solvd.laba.ermakovich.hu.mongo;
 
 import org.springframework.data.mongodb.repository.ReactiveMongoRepository;
 import reactor.core.publisher.Mono;
-import solvd.laba.ermakovich.hu.service.aggregate.DoctorAggregate;
+import solvd.laba.ermakovich.hu.aggregate.DoctorAggregate;
 
 import java.util.UUID;
 
 /**
  * @author Ermakovich Kseniya
  */
-public interface DoctorRepository extends ReactiveMongoRepository<DoctorAggregate, Long> {
+public interface DoctorRepository extends ReactiveMongoRepository<DoctorAggregate, String> {
 
-    Mono<Boolean> existsByEmail(String email);
+    Mono<Boolean> existsByDoctor_Email(String email);
```

Comment:
please follow the java code conventions. [here](https://www.oracle.com/technetwork/java/codeconventions-150003.pdf) it is

Reply:
I thought that query with field inside entity works only with '_'. My mistake